src/Our.Umbraco.BackofficeThemes/Properties/VersionInfo.cs

```diff
@@ -0,0 +1,19 @@
+//------------------------------------------------------------------------------
+// <auto-generated>
+//     Dieser Code wurde von einem Tool generiert.
+//     Laufzeitversion:4.0.30319.42000
+//
+//     Änderungen an dieser Datei können falsches Verhalten verursachen und gehen verloren, wenn
+//     der Code erneut generiert wird.
+// </auto-generated>
+//------------------------------------------------------------------------------
+
+using System;
+using System.Reflection;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+
+[assembly: AssemblyVersion("0.1-alpha.*")]
```

Comment:
I'm not sure that this compiles? The `AssemblyVersion` attribute should require a valid version number - with major.minor.patch numerics.

`AssemblyInformationalVersion` is different... you can have any old string in there - it's purely for presentational purposes.